DTS-STN/Service-Canada-Labs
# All Updates Page + Updated Homepage + Updated Article and Project Pages

### [Create "All Updates" Page](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities?workitem=215235)

### [Modify Homepage template](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities?workitem=215234)

Figma Designs: https://www.figma.com/design/ke3SrZNGMP3AjwzMMuEgbo/[current]-BDM-DECD-SC-Labs?node-id=19239-43264&t=g2S8D7ARF4lotf5Q-0

Massive PR because the git history is messed up right now. Will be working on a cleaner git workflow in the future.

This PR includes:

- The new All Updates page
- Updated Homepage to match new designs
- Updated project pages and article pages to match new designs (article pages still need the "Explore this project section" and some AEM content still needs to be hooked up)
- Updated tests

Work that still needs to be done:
- Some AEM content still needs to be hooked up such as headings in some places (this PR will be followed by that work)

## Test Instructions

1. yarn build + yarn start
2. Navigate to homepage, see that everything renders as expected, links work and it matches design
3. Navigate to project and article pages, see that they render, links work and they matches design (excluding "Explore this project section")
